cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -19,6 +19,10 @@ pub struct EdgeAddRemovePlugin {
 
     #[default(DEFAULT_REMOVE_ALL_EDGES_VALUE.to_string())]
     pub remove_all_edges_value: String,
+
+    /// If true causes the removal of all processed metadata from the releases.
+    #[default(false)]
+    pub remove_consumed_metadata: bool,
```

Comment:
Do we need to make this tuneable?  Can we just always do it?

Reply:
It can be helpful for debugging to leave them in, so see what annotations are on the release. Also for now it's still off by default, which is compatible with the existing deployments.

Reply:
Can this be used to force the plugin to discard previous processed metadata and start with fresh metadata scanning

Reply:
This controls whether the metadata which is used by this plugin is removed after processing or not. As an example, if there is a `previous.remove` with the value `1.2.3` metadata on a release, this plugin will remove the edge, in the graph. If `remove_consumed_metadata` is `true` in the plugin configuration, it will also remove the metadata entry with the key `previous.remove`.